openwisp_ipam/migrations/0005_default_groups_permissions.py

```diff
@@ -6,7 +7,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('openwisp_users', '0004_default_groups'),
+        ('openwisp_users', '0011_user_first_name_150_max_length'),
```

Comment:
@atb00ker @nemesisdesign I wanna remove `openwisp_users` as a dependency here but `openwisp_users` is migrated after `openwisp_ipam` if I do so.
How this can be fixed?

Reply:
use `swappable_dependency`, read here: https://github.com/openwisp/django-swappable-models#migration-scripts